src/sentry/api/serializers/rest_framework/sentry_app.py

```diff
@@ -26,10 +29,19 @@ def validate(self, data):
             ))
 
 
+class SchemaField(serializers.WritableField):
+    def validate(self, data):
+        try:
+            validate(data)
```

Comment:
We should probably import this as `validate_schema` or something so it doesn't collide with the method it's called in.